### ISSUE TITLE
How can I change the left nav and subitem colors? 

### ISSUE THREAD

**Issue by mattortiz** [ant-design/ant-design-pro#7516]
### 🧐 How can I change the left nav, groups, and subitem colors? 

Thank you, Ant team, for open sourcing this UI solution! I am trying to change the nav and some simple theming. I've reviewed [issue 2334](https://github.com/ant-design/ant-design-pro/issues/2334). However, the app.css (only found in node_modules/antd/dist/) described in the issue has (presumably) all styles in it already. I see the .ant-menu... styles are there, but changes do not override at build/runtime when changed. 

Can you point us at the right place to do such overrides? A listing of typical overridden styles/elements would be very much appreciated as well; e.g.: menus, backgrounds, etc. Note: I'm familiar with the Ant.design theme overrides, but given the issue referenced, it'd be great to get your recommendations. 

I'm new to the UI solution, so forgive me if I've missed this in existing documentation. 

![Screen Shot 2020-10-23 at 6 30 06 AM](https://user-images.githubusercontent.com/128472/97011636-ccc8f180-14fb-11eb-9cbd-b18c43973683.png)

I don't see the same settings in the UI as shown in Ant Pro Layout [docs](https://github.com/ant-design/ant-design-pro-layout), so it is likely I'm just missing something simple: 

![Screen Shot 2020-10-23 at 7 01 41 AM](https://user-images.githubusercontent.com/128472/97013298-bf146b80-14fd-11eb-98b1-3a9d5c5f6beb.png)

**Comment by mattortiz** [comment]
Ok, I found it. I was able to find the CSS and use the method described in Ant.design documentation here. 

My app/config/config.js (JS version) now looks like this to override the sidebar menu. 

  theme: {
    // ...darkTheme,
    'primary-color': defaultSettings.primaryColor,
    '@layout-header-background': '#233A57',
  },

And the result (yes, more to do) is: 

![Screen Shot 2020-10-23 at 2 09 25 PM](https://user-images.githubusercontent.com/128472/97054359-82fefc00-1539-11eb-944b-b2e2c9daa226.png)

Closing but hope it's helpful. Note: this was not an issue/or knowledge gap on Ant Design Pro. It was me needing to find the right CSS element and put it in the config to override it. 

Matt
